owncloudApp/build.gradle

```diff
@@ -92,8 +92,8 @@ android {
 
         testInstrumentationRunner "com.owncloud.android.utils.OCTestAndroidJUnitRunner"
 
-        versionCode = 40000000
-        versionName = "4.0"
+        versionCode = 41000000
```

Comment:
Please, be compliant with the regular format for our beta versions:

```
In versionCode, we will increase the latest digits, with the beta version
In versionName, we will add beta and use it this way: "M.m.p-beta.b".
In this case, since minor version is 0, it is omitted.

versionCode = 21800301
versionName = "2.19-beta.1"
```

versionCode should be `40000001`